manifests/00-cluster-role.yaml

```diff
@@ -8,6 +8,7 @@ rules:
   - ""
   resources:
   - namespaces
+  - secrets
```

Comment:
Ouch

Reply:
Yeah, but how can you get around that?  That's the problem with this approach.

At least it's not the router that can read secrets.

Reply:
True. Only way I know of to get around it is to let the CVO create the `openshift-ingress` namespace and a role binding in that namespace for the operator SA.

Reply:
After further discussion, we decided to defer this issue for now.